dist/tools/elf2uf2/elf2uf2.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# $1 contains $(RIOTTOOLS) from makefile
+# $2 contains generated elf file - $FLASHFILE from makefile
+
+$1/elf2uf2/elf2uf2 $2 NEW.UF2
+
+mv NEW.UF2 /media/$USER/RPI-RP2/.
```

Comment:
```suggestion
mv NEW.UF2 "/media/$USER/RPI-RP2/"
```

The `.` refers to "current dir", I think it is not needed. I also don't think that user names with spaces are valid, but making the CI happy doesn't hurt.